back-end/services/projects.service.js

```diff
@@ -0,0 +1,55 @@
+import ProjectsDAO from "../dao/projectsDAO.js";
```

Comment:
Nice work on the projects service and removing the redundant getFilteredProjects method. I also removed this method in 19-dashboard-ui, so we need to keep this in mind when merging that branch.